src/main/thrift/parquet.thrift

```diff
@@ -211,6 +204,93 @@ struct Statistics {
    4: optional i64 distinct_count;
 }
 
+/** Empty structs to use as logical type annotations */
+struct StringType {}
+struct MapType {}
+struct ListType {}
+struct EnumType {}
+struct DateType {}
+
+/**
+ * Logical type to annotate a column that is always null.
+ *
+ * Sometimes when discovering the schema of existing data values are always
```

Comment:
I found this sentence a bit tricky to parse. Can you add a comma to make it more clear?

Reply:
I added some clarification here. Thanks for the suggestion.